examples/jaeger-skaffold-trace/README.md

```diff
@@ -0,0 +1,37 @@
+### Example: Skaffold Command Tracing w/ Jaeger
+
+In this example:
+
+* Use skaffold to deploy a jaeger local/remote jaeger instance
```

Comment:
Jaeger and Skaffold should be capitalized since that's the conventional form both use.  Use `skaffold` to refer to the command.

```suggestion
* Use Skaffold to deploy a local/remote Jaeger instance
```

Reply:
Done

Reply:
Updated this in all relevant places for the example